### ISSUE TITLE
String and Text fields fail OpenAPI Schema validation

### ISSUE THREAD

**Issue by johnthagen** [collerek/ormar#291]
**Describe the bug**

Using `String` and `Text` fields results in Schema validation errors when run against [`openapi-generator`](https://openapi-generator.tech/).

**To Reproduce**

```python
import databases
from fastapi import FastAPI
import ormar
import sqlalchemy

DATABASE_URL = "sqlite:///db.sqlite"
database = databases.Database(DATABASE_URL)
metadata = sqlalchemy.MetaData()


class BaseMeta(ormar.ModelMeta):
    metadata = metadata
    database = database


class Author(ormar.Model):
    class Meta(BaseMeta):
        tablename = "authors"

    id: int = ormar.Integer(primary_key=True)
    name: str = ormar.String(max_length=100)
    contents: str = ormar.Text()


app = FastAPI()


@app.get("/", response_model=Author)
async def root(author: Author):
    return author
```

Running OpenAPI generator 5.2.0 against the schema generated by FastAPI fails:

```bash
$ npx @openapitools/openapi-generator-cli generate \
>     --input-spec http://localhost:8000/openapi.json \
>     --output src/api \
>     --generator-name typescript-axios
Exception in thread "main" org.openapitools.codegen.SpecValidationException: There were issues with the specification. The option can be disabled via validateSpec (Maven/Gradle) or --skip-validate-spec (CLI).
 | Error count: 2, Warning count: 0
Errors: 
        -attribute components.schemas.Author.strip_whitespace is unexpected
        -attribute components.schemas.Author.allow_blank is unexpected

        at org.openapitools.codegen.config.CodegenConfigurator.toContext(CodegenConfigurator.java:546)
        at org.openapitools.codegen.config.CodegenConfigurator.toClientOptInput(CodegenConfigurator.java:573)
        at org.openapitools.codegen.cmd.Generate.execute(Generate.java:433)
        at org.openapitools.codegen.cmd.OpenApiGeneratorCommand.run(OpenApiGeneratorCommand.java:32)
        at org.openapitools.codegen.OpenAPIGenerator.main(OpenAPIGenerator.java:66)
```

**Expected behavior**

OpenAPI Generator should not throw a schema validation error.

**Versions:**
 - Database backend used: sqlite
 - Python version: 3.9.6

```bash
$ pip list
Package           Version
----------------- -------
aiosqlite         0.17.0
asgiref           3.4.1
click             8.0.1
databases         0.4.3
fastapi           0.67.0
h11               0.12.0
httptools         0.2.0
ormar             0.10.15
pep517            0.11.0
pip               21.2.1
pip-tools         6.2.0
pydantic          1.8.2
python-dotenv     0.19.0
PyYAML            5.4.1
setuptools        57.0.0
SQLAlchemy        1.3.23
starlette         0.14.2
tomli             1.1.0
typing-extensions 3.7.4.3
uvicorn           0.14.0
uvloop            0.15.3
watchgod          0.7
websockets        9.1
wheel             0.36.2
```

**Additional context**

It seems as though `allow_blank` and `strip_whitespace` are slipping into the attributes of the generated schema:

<img width="597" alt="Screen Shot 2021-07-28 at 10 23 32 AM" src="https://user-images.githubusercontent.com/10340167/127339375-cbb8f500-9a61-48d3-8b73-93e6d01ebdaa.png">

**Comment by johnthagen** [comment]
This bug seems to have manifested with [FastAPI 0.66.0](https://github.com/tiangolo/fastapi/releases/tag/0.66.0). Installing FastAPI 0.65.3 does not trigger this bug, but every version after 0.66.0 does.

**Comment by johnthagen** [comment]
Perhaps these need to be prefixed with `x-`?

https://github.com/tiangolo/fastapi/pull/1429#issuecomment-889315574

**Comment by collerek** [comment]
@johnthagen thanks for the investigation, will take a look at this

**Comment by collerek** [comment]
Fixed in #298
